product_docs/docs/pem/8/installing_pem_server/pem_server_inst_windows/pem_server_and_httpd_on_same_host.mdx

```diff
@@ -46,8 +46,8 @@ The easiest PEM server installation configuration consists of a PEM backend data
 1.  The wizard is now ready to install the PEM server. Select **Next** to continue with the installation.
 
   During the installation process, the installer copies files to the system and sets up the database and web services required to run PEM. When the installation completes, a confirmation indicates that:
-  -  The web service was configured.
-  -  The web service is listening on port 8443.
-  -  The `pem` database was created and configured.
+    -  The web service was configured.
+    -  The web service is listening on port 8443. (You can change the default 8443 port as per your requirement. To change the port see [change the default port](/pem/latest/changing_default_port)).
```

Comment:
remove this since windows steps are still pending

Reply:
removed and committed